thundra/thundra_agent.py

```diff
@@ -142,70 +146,66 @@ def wrapper(event, context):
 
     call = __call__
 
-    def start_debugger_tracing(self):
+    def initialize_debugger(self):
         try:
             import ptvsd
-            ptvsd.enable_attach(address=("localhost", config.debugger_broker_port()))
-
-            self.debugger_process = subprocess.Popen([
-                "/opt/socat",
-                "TCP:localhost:{}".format(config.debugger_broker_port()),
-                "TCP:{}:{}".format(config.debugger_broker_host(), config.debugger_broker_port())]
-              )
-
-            ptvsd.wait_for_attach(config.debugger_max_wait_time()/1000)
-            self.wait_for_debugger()
-            ptvsd.tracing(True)
-
+            ptvsd.enable_attach(address=("localhost", config.debugger_port()))
```

Comment:
Why we are enabling attach at the both of `initialize_debugger` and `start_debugger_tracing`. I know `initialize_debugger` runs only once at first invocation but in this case seems that `start_debugger_tracing` is also called and it also enables attach. If `start_debugger_tracing` already enables attaching, why we need `initialize_debugger`